misk/src/main/kotlin/misk/metrics/backends/stackdriver/StackDriverBackendModule.kt

```diff
@@ -0,0 +1,32 @@
+package misk.metrics.backends.stackdriver
+
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
+import com.google.api.client.json.jackson2.JacksonFactory
+import com.google.api.services.monitoring.v3.Monitoring
+import com.google.common.util.concurrent.Service
+import com.google.inject.AbstractModule
+import com.google.inject.Provides
+import com.google.inject.Singleton
+import misk.config.AppName
+import misk.inject.addMultibinderBinding
+import misk.inject.to
+
+class StackDriverBackendModule : AbstractModule() {
+    override fun configure() {
+        bind(StackDriverSender::class.java)
+                .to(StackDriverBatchedSender::class.java)
+                .`in`(Singleton::class.java)
```

Comment:
You can use `inSingleton()` which we define in `Guice.kt` to avoid having the ugly quotes.